src/sstruct_mv/sstruct_matmult.c

```diff
@@ -66,7 +66,7 @@ hypre_SStructMatmult( HYPRE_Int             nmatrices,
    HYPRE_Int                m, s;
    HYPRE_Int                part;
 
-#if DEBUG_MATCONV
+#if defined(HYPRE_DEBUG) && defined(DEBUG_MATCONV)
```

Comment:
Why should this be changed in this way?

Reply:
Hi Rob, I was going back and forth between release and debug modes and wanted to run these statements only in debug mode. So, before, every time I compiled the code in release mode, I needed to go back to the code and undefine `DEBUG_MATCONV`. Now, I can keep such variable always defined and the statements bellow this line will run only when hypre is configured in debug mode. It was basically a smart way to have these debugging statements without needing to change code every time I changed compile mode.